src/components/Carousel/CarouselDot.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { useInView } from 'react-intersection-observer';
+
+type Props = {
```

Comment:
why the type and not an interface?

Reply:
should be an interface with comments so we get the documentation in storybook.

Reply:
this is internal to to `CarouselDots`, which is the exposed to consumers, so there's not actually a storybook for it

i can update to interface tho